<!doctype html>
<html>
<head>
  <title>Software of NGS</title>
</head>
<body>
<header class="entry-header" style="box-sizing: border-box; width: 620px; margin-bottom: 2.3rem; color: rgb(77, 59, 24); font-family: 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', Geneva, Verdana, sans-serif; font-size: 13px; line-height: 27.6px; ">
<h1 class="entry-title" style="box-sizing: border-box; clear: both; font-family: Lato, sans-serif; font-weight: 400; line-height: 1.5; margin: 0px; font-size: 3rem; color: rgb(113, 89, 44); text-transform: uppercase; width: 613.797px;">NGS SOFTWARE</h1>
</header>

<div class="entry-content" style="box-sizing: border-box; width: 620px; color: rgb(77, 59, 24); font-family: 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', Geneva, Verdana, sans-serif; font-size: 13px; line-height: 27.6px;">
<p style="box-sizing: border-box; margin: 0px 0px 2.3rem;"><span style="box-sizing: border-box; text-decoration: underline;"><b style="box-sizing: border-box;">Next-gen sequence alignment and&nbsp;RNA-seq analysis tools</b></span></p>

<ol style="box-sizing: border-box; margin: 1.6rem 0px; padding: 0px 0px 0px 4rem;">
  <li style="box-sizing: border-box;"><a href="http://bowtie-bio.sf.net/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s3" style="box-sizing: border-box;">Bowtie</span></a>&nbsp;is an ultrafast system for aligning short reads from next-generation sequencers to the human genome and any other genome.&nbsp;&nbsp;<a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Bowtie2</span></a>, which supports gapped alignments, longer reads, and is equally fast, appeared in late 2011. The Bowtie project is led by&nbsp;<a href="http://cs.jhu.edu/~langmea" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Ben Langmead</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://tophat.cbcb.umd.edu/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s3" style="box-sizing: border-box;">Tophat</span></a>&nbsp;is a fast splice junction mapper for RNA-Seq reads.&nbsp; TopHat is independent of annotation, meaning it can find novel exons and splice sites even if they are missing from standard gene annotation.&nbsp; TopHat was originally developed by Cole Trapnell.<b style="box-sizing: border-box;">TopHat2</b>&nbsp;was developed primarily by&nbsp;<a href="http://www.ccb.jhu.edu/people/infphilo/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);">Daehwan Kim</a>.</li>
  <li style="box-sizing: border-box;"><a href="http://www.ccb.jhu.edu/software/hisat/index.shtml" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);" target="_blank">HISAT</a>&nbsp;is a very fast splice junction mapper, a successor to TopHat that is equally accurate and up to 50 times faster. The HISAT project is led by&nbsp;<a href="http://www.ccb.jhu.edu/people/infphilo/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);" target="_blank">Daehwan Kim</a>.</li>
  <li style="box-sizing: border-box;"><a href="https://ccb.jhu.edu/software/stringtie/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">StringTie</span></a>&nbsp;is a new and fast transcript assembler and abundance estimator for RNA-seq data. Designed as a successor&nbsp;to Cufflinks, StringTie assembles transcripts from the alignments produced by TopHat, identifying novel isoforms and estimating expression levels for all transcripts. The StringTie project is led by&nbsp;<a href="http://ccb.jhu.edu/people/mpertea/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Ela Pertea</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://cole-trapnell-lab.github.io/cufflinks/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s3" style="box-sizing: border-box;">Cufflinks</span></a>&nbsp;assembles the reads from an RNA-seq experiment, producing full-length transcripts in multiple isoforms, quantitating the levels of expression of each gene and each isoform.&nbsp; The Cufflinks project is led by former student&nbsp;<a href="http://cole-trapnell-lab.github.io/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Cole Trapnell</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://mummer.sourceforge.net/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">MUMmer</span></a>&nbsp;is a system for aligning whole genomes, chromosomes, and other very long DNA sequences.&nbsp; It includes the Nucmer and Promer alignment tools. MUMmer has been in continuous usage for &gt;15 years, and is still actively used and supported by the lab.</li>
  <li style="box-sizing: border-box;"><span class="s4" style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/diamund/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">DIAMUND</span></a>&nbsp;</span>is an efficient algorithm for variant detection that compares DNA sequences directly to one another, without aligning them to the reference genome. When used on exome sequences from family trios, or to compare normal and diseased samples from the same individual, it produces a dramatically smaller list of candidate mutations than previous methods. Original developers: Steven Salzberg and Ela Pertea.</li>
  <li style="box-sizing: border-box;"><span class="s4" style="box-sizing: border-box;"><a href="http://tophat-fusion.sourceforge.net/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">TopHat-Fusion</span></a></span>&nbsp;is an enhanced version of TopHat with the ability to align reads across chromosomal fusion points, which results from the breakage and re-joining of different chromosomes, a common event in some tumors. Original developer: Daehwan Kim.</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/EDGE-pro/index.shtml" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">EDGE-pro</span></a>&nbsp;aligns and quantitates transcript data from bacterial and archaeal RNA-seq experiments. Original developer: Tanja Magoc.</li>
</ol>

<p class="p1" style="box-sizing: border-box; margin: 0px 0px 2.3rem;"><span style="box-sizing: border-box; text-decoration: underline;"><span class="s1" style="box-sizing: border-box;"><b style="box-sizing: border-box;">Genome Assembly</b></span></span></p>

<ol style="box-sizing: border-box; margin: 1.6rem 0px; padding: 0px 0px 0px 4rem;">
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/quake/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Quake</span></a>&nbsp;is an error-correction package that&nbsp;detects and correct substitution sequencing errors in whole-genome sequencing data sets with deep coverage, primarily for next-generation sequencing projects. Original developer: David Kelley.&nbsp;&nbsp;<a href="http://genomebiology.com/2010/11/11/R116/abstract" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Read the paper</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/FLASH/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">FLASH</span></a>, Fast Length Adjustment of SHort reads, is a very fast program to merge paired-end reads that were sequenced from fragments that are shorter than twice the read length. Original developer: Tanja Magoc.&nbsp;<a href="http://bioinformatics.igm.jhmi.edu/salzberg/docs/Bioinformatics-2011-Magoc-Salzberg-FLASH-reprint.pdf" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Read the paper</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://amos.sourceforge.net/docs/pipeline/AMOScmp.html" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">AMOScmp</span></a>&nbsp;is a comparative genome assembler, which uses one genome as a reference on which to assemble another, closely related species.&nbsp; Original developers: Mihai Pop and Adam Phillippy.&nbsp;<a href="http://ccb.jhu.edu/people/salzberg/docs/AMOScmp-reprint.pdf" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Read the paper here</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://sourceforge.net/apps/mediawiki/amos/index.php?title=Minimus" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Minimus</span></a>&nbsp;is a small, lightweight assembler for small jobs such as assembling a viral genome, assembling a set of reads from a single gene, or other tasks that don&rsquo;t require a large-genome assembler. Original developer: Daniel Sommer.&nbsp;&nbsp;<a href="http://www.biomedcentral.com/1471-2105/8/64" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Read the paper</span></a>.</li>
  <li style="box-sizing: border-box;">The&nbsp;<a href="http://amos.sourceforge.net/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s3" style="box-sizing: border-box;">AMOS Assembler project</span></a>&nbsp;is a set of tools, libraries, and freestanding genome assemblers, all open source. AMOS is also an open consortium that we started at TIGR, and that now includes multiple institutions.</li>
  <li style="box-sizing: border-box;"><a href="http://amos.sourceforge.net/hawkeye/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Hawkeye</span></a>, a flexible graphical interface to genome assemblies from a variety of assemblers.&nbsp; Original developers: Mike Schatz and Adam Phillippy.&nbsp;<a href="http://genomebiology.com/2007/8/3/R34" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Read the paper</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://sourceforge.net/apps/mediawiki/amos/index.php?title=Bambus" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Bambus</span></a>&nbsp;was the first publicly available, standalone genome assembly scaffolder. It orders and orients contigs into scaffolds based on various types of linking information.&nbsp;&nbsp;<a href="http://cbcb.umd.edu/~mpop" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Mihai Pop&rsquo;s lab</span></a>&nbsp;subsequently released&nbsp;<a href="http://sourceforge.net/apps/mediawiki/amos/index.php?title=Bambus2" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Bambus2</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://sourceforge.net/apps/mediawiki/amos/index.php?title=AutoEditor" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">AutoEditor</span></a>, a tool for correcting sequencing and basecaller errors using sequence assembly and chromatogram data (for older, capillary-based sequencers). On average AutoEditor corrects 80% of erroneous base calls, with an accuracy of 99.99%.&nbsp; Original developers: Pavel Gajer and Mike Schatz.&nbsp;<a href="http://nar.oxfordjournals.org/content/32/2/562.full" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Read the paper</span></a>.</li>
</ol>

<p class="p1" style="box-sizing: border-box; margin: 0px 0px 2.3rem;"><span style="box-sizing: border-box; text-decoration: underline;"><span class="s1" style="box-sizing: border-box;"><b style="box-sizing: border-box;">Computational Gene Finding and Metagenomics</b></span></span></p>

<ol style="box-sizing: border-box; margin: 1.6rem 0px; padding: 0px 0px 0px 4rem;">
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/glimmer" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Glimmer</span></a>&nbsp;uses interpolated Markov models (IMMs) to find genes in microbial DNA. Used around the world for thousands of genomes.&nbsp; Originally developed by Art Delcher and Steven Salzberg.</li>
  <li style="box-sizing: border-box;"><a href="https://ccb.jhu.edu/software/kraken/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Kraken</span></a>&nbsp;is a very fast system for identifying the species represented by short (or long) DNA sequences, usually obtained through microbiome or metagenomic studies. The Kraken project is led by former Ph.D. student&nbsp;<a href="http://ccb.jhu.edu/people/dwood/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Derrick Wood</span></a>.</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/phymmbl/index.shtml" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Phymm and PhymmBL</span></a>, first released in 2009, are systems for classifying short DNA sequences from metagenomics projects, labelling them with their likely species name. Originally developed by Arthur Brady.</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/jigsaw/jigsaw-3.2.6.tar.gz" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">JIGSAW,</span></a>&nbsp;a program that predicts gene models using the output from multiple sources of evidence, including other gene finders, Blast searches, and other alignment data. Originally developed by Jonathan Allen.</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/glimmerhmm/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">GlimmerHMM,</span></a>&nbsp;an interpolated Markov Model system for finding genes in many eukaryotes, including&nbsp;<i style="box-sizing: border-box;">P. falciparum, A. thaliana,&nbsp;</i>rice&nbsp;<i style="box-sizing: border-box;">(O. sativa),&nbsp;</i>mosquito&nbsp;<i style="box-sizing: border-box;">(A. aegypti), B. malayi, C. neoformans</i>, and others. Originally developed by&nbsp;<a href="http://ccb.jhu.edu/people/mpertea/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);" target="_blank">Mihaela Pertea</a>.</li>
  <li style="box-sizing: border-box;"><a href="http://www.genezilla.org/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">GeneZilla</span></a>, a generalized HMM for eukaryotic gene finding developed by&nbsp;<a href="http://www.duke.edu/~bmajoros/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Bill Majoros</span></a>, a former Salzberg lab member (when the lab was at TIGR).</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/genesplicer/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">GeneSplicer,</span></a>&nbsp;a fast system for detecting splice sites in genomic DNA of various eukaryotes. Originally developed by&nbsp;<a href="http://ccb.jhu.edu/people/mpertea/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);" target="_blank">Mihaela Pertea</a>.</li>
</ol>

<p class="p1" style="box-sizing: border-box; margin: 0px 0px 2.3rem;"><span style="box-sizing: border-box; text-decoration: underline;"><span class="s1" style="box-sizing: border-box;"><b style="box-sizing: border-box;">Transcription terminators, operons, and motif analysis tools</b></span></span></p>

<ol style="box-sizing: border-box; margin: 1.6rem 0px; padding: 0px 0px 0px 4rem;">
  <li style="box-sizing: border-box;"><a href="http://transterm.ccb.jhu.edu/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">TransTermHP</span></a>&nbsp;(<span class="s3" style="box-sizing: border-box;">updated in 2010</span>), a program that finds rho-independent transcription terminators in bacterial genomes. Originally developed in 2000 by Maria Ermolaeva.&nbsp; Re-designed and re-implemented in 2007 by Carl Kingsford.</li>
  <li style="box-sizing: border-box;"><a href="http://operondb.ccb.jhu.edu/cgi-bin/operons.cgi" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">OperonDB</span></a>&nbsp;(<span class="s3" style="box-sizing: border-box;">update in progress, 2015</span>), results from our operon-finding software on a large number of prokaryotic genomes.&nbsp;Described in&nbsp;<span class="s2" style="box-sizing: border-box;"><a href="http://nar.oxfordjournals.org/content/37/suppl_1/D479.abstract?keytype=ref&amp;ijkey=5lkSgVyZSaFkRIt" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);">OperonDB: a comprehensive database of predicted operons in microbial genomes</a>&nbsp;(Pertea et al. 2009)</span>. Originally developed in 2001 by Maria Ermolaeva.&nbsp; Redesigned and re-implemented in 2008 by Mihaela Pertea.</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/ELPH/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">ELPH</span></a>, a motif finder that can find ribosome binding sites, exon splicing enhancers, or regulatory sites. Original developer: Mihaela Pertea.</li>
  <li style="box-sizing: border-box;"><a href="http://www.cbcb.umd.edu/software/SeeEse/" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">SeeESE</span></a>, an online tool for identifying exon splicing enhancers (ESEs) in Arabidopsis, Drosophila, and other species. Originally developers: Mihaela Pertea and Steven Mount.</li>
  <li style="box-sizing: border-box;">Skewed oligomers from bacterial and archaeal genomes (described in&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pubmed/9795135" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Salzberg et al.,<i style="box-sizing: border-box;">Gene</i>&nbsp;217:1-2, 1998</span></a>). &nbsp;<a href="http://bioinformatics.igm.jhmi.edu/salzberg/docs/skew/oligoskew.tar" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s2" style="box-sizing: border-box;">Get the source code</span></a>.</li>
</ol>

<p class="p1" style="box-sizing: border-box; margin: 0px 0px 2.3rem;"><span style="box-sizing: border-box; text-decoration: underline;">Machine learning systems, pre-1995 and pre-computational biology</span></p>

<ul style="box-sizing: border-box; margin: 1.6rem 0px; padding: 0px 0px 0px 4rem; list-style-type: square;">
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/oc1/oc1.tar.gz" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s1" style="box-sizing: border-box;">The OC1 decision tree system</span></a>.&nbsp; Originally developed by S.K. Murthy.</li>
  <li style="box-sizing: border-box;"><a href="http://ccb.jhu.edu/software/pebls/pebls.tar.gz" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><span class="s1" style="box-sizing: border-box;">The PEBLS memory-based reasoning system</span></a>.&nbsp; Originally developed by Scott Cost.</li>
</ul>

<p style="box-sizing: border-box; margin: 0px 0px 2.3rem;"><a href="https://salzberglab.files.wordpress.com/2015/02/stringtie_step2-001.png" style="box-sizing: border-box; color: rgb(3, 101, 100); text-decoration: none; transition: none; border-bottom-width: 1px; border-bottom-style: solid; border-bottom-color: rgba(0, 0, 0, 0.0980392);"><img alt="" class="alignnone wp-image-197 " height="170" src="https://salzberglab.files.wordpress.com/2015/02/stringtie_step2-001.png?w=671&amp;h=170" style="box-sizing: border-box; border: 0px; vertical-align: middle; margin: 0.5rem 0px; height: auto; max-width: 100%;" width="671" /></a></p>
</div>
</body>
</html>

